src/modals/EditOrderModal/EditOrderModal.js

```diff
@@ -69,6 +69,18 @@ const processUpdateOrder = (order, id) => ({
   flags: calculateFlags(order),
 })
 
+const processAOArgs = (args) => {
+  const updArgs = { ...args }
+  updArgs.action = updArgs.amount < 0 ? 'Sell' : 'Buy'
```

Comment:
can you a task for small refactoring, to have util to decide Sell/Buy mode based on amount. I see we already have this logic at other 3 places in codebase. Can be addressed in separate PR.